oneanddone/functional_tests/pages/base.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Yes, these are identical to the tests in oneanddone-tests, which I will remove once this process is complete, so no need to review the tests themselves.